src/model/AirTerminalDualDuctVAV.cpp

```diff
@@ -159,6 +173,32 @@ namespace model {
       OS_ASSERT(result);
     }
 
+    bool AirTerminalDualDuctVAV_Impl::setMinimumAirFlowTurndownSchedule(Schedule& schedule) {
+      bool result = setSchedule(OS_AirTerminal_DualDuct_VAVFields::MinimumAirFlowTurndownScheduleName, "AirTerminalSingleDuctVAVReheat",
+                                "Minimum Air Flow Turndown", schedule);
+      return result;
+    }
```

Comment:
* [ ] TODO: For this file, I believe you forgot to modify getScheduleTypeKeys. on line 79